docs/_docs/framework/capabilities/manage/practice-ops.md

```diff
@@ -0,0 +1,116 @@
+ms.topic: conceptual
+ms.service: finops
+ms.reviewer: micflan
+-->
+
+<span class="fs-9 d-block mb-4">FinOps Practice Operations</span>
+This article helps you understand the FinOps Practice Operations capability within the FinOps Framework and how to implement that in the Microsoft Cloud.
+{: .fs-6 .fw-300 }
+
+<details open markdown="1">
+  <summary class="fs-2 text-uppercase">On this page</summary>
+
+- [▶️ Getting started](#️-getting-started)
+- [🏗️ Building on the basics](#️-building-on-the-basics)
+- [🍎 Learn more at the FinOps Foundation](#-learn-more-at-the-finops-foundation)
+- [⏩ Next steps](#-next-steps)
+- [ms.reviewer: micflan](#msreviewer-micflan)
+
+</details>
+
+---
+
+<a name="definition"></a>
+**The FinOps Practice and Operations capability focuses on building a robust FinOps team, defining clear responsibilities, and integrating FinOps practices into organizational processes to manage cloud costs effectively.**
+{: .fs-6 .fw-300 }
+
+Establish a dedicated FinOps team with the necessary skills, define and implement a FinOps strategy, foster stakeholder relationships, and develop a growth and maturity plan for the team.
+
+<br>
+
+## ▶️ Getting started
+
+To get started with this capability, follow these steps:
+
+- Establish clear roles and responsibilities within the FinOps practice. Initially, this may involve one or two individuals managing the FinOps responsibilities until a [dedicated team](docs\_docs\framework\capabilities\manage\structure.md) can be formed.
+- Define a clear FinOps strategy and integrate it with [organizational processes](docs\_docs\framework\capabilities\manage\intersecting-disciplines.md).
+- Build strong relationships with stakeholders and promote a [FinOps culture](docs\_docs\framework\capabilities\manage\culture.md) within the organization.
+- Develop a growth and maturity plan for the team's skills and capabilities.
+
+<br>
+
+## 🏗️ Building on the basics
+
+- Form a dedicated FinOps team with diverse skills and adequate resources.
+- Provide continuous [training and skill development](docs\_docs\framework\capabilities\manage\education.md) for team members.
+- Regularly review and refine the FinOps strategy to ensure alignment with organizational objectives.
+- Foster a culture of collaboration and communication across departments to enhance FinOps practices.
+
+
+<br>
+
+## 🍎 Learn more at the FinOps Foundation
+
+This capability is part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [FinOps Practice and Operations](https://www.finops.org/framework/capabilities/finops-practice-operations/) article in the FinOps Framework documentation.
+
+<br>
+
+## ⏩ Next steps
+
+Related FinOps capabilities:
+
+- [Assessment](../manage/assessment.md)
+- [Benchmarking](../optimize/benchmarking.md)
+
```

Comment:
For related capabilities, should we add FinOps Tools & Services and FinOps Education & Enablement as referenced in the Inputs and Outputs section of the FinOps Foundation/ FinOps Practice Operations Capability https://www.finops.org/framework/capabilities/finops-practice-operations/. 

```suggestion
- [FinOps Tools & Services](docs/_docs/framework/capabilities/manage/education.md)
- [FinOps Education & Enablement](docs/_docs/framework/capabilities/manage/tools-and-services.md)
```

Note: Please verify links if suggestion is added into document.

Reply:
Education and enablement makes sense. I'm not aware of any dedicated tools for FinOps practice operations, so we can probably leave that one out for now. We can add it later if there is something specific that I'm missing.
```suggestion
- [FinOps education and enablement](./education.md)

```